system-int-test/test/cmr/system_int_test/ingest/collection_umm_spec_validation_test.clj

```diff
@@ -216,6 +218,33 @@
       ["Instruments must be unique. This contains duplicates named [I1]."]))
 
   (testing "Spatial validation"
+    (testing "Spatial enums"
+      (let [geometry
+            {:Geometry (umm-cmn/map->GeometryType
+                        {:CoordinateSystem "CARTESIAN"
+                         :Lines [(umm-cmn/map->LineType
+                                   {:Points [(umm-cmn/map->PointType {:Longitude 180 :Latitude 0})
+                                             (umm-cmn/map->PointType {:Longitude -180 :Latitude 0})]})]})}]
+        (testing "Invalid GranuleSpatialRepresentation enum"
+          (assert-invalid-spatial "INVALID_GRANULE_SPATIAL_REPRESENTATION"
+                                  geometry
+                                  ["SpatialExtent"]
+                                  ["Value (\"INVALID_GRANULE_SPATIAL_REPRESENTATION\") not found in enum (possible values: [\"CARTESIAN\",\"GEODETIC\",\"NO_SPATIAL\",\"ORBIT\"])"]
+                                  {:format :iso19115}))
+        (testing "Valid GranuleSpatialRepresentation enum"
+          (assert-valid-spatial "CARTESIAN"
+                                geometry
+                                {:format :iso19115}))
+        (testing "Invalid CoordinateSystem enum"
+          (assert-invalid-spatial "CARTESIAN"
+                                  (assoc-in geometry [:Geometry :CoordinateSystem] "INVALID_COORDINATE_SYSTEM")
```

Comment:
Why do you need to pass the valid "CARTESIAN" and associate the invalid one, rather than passing in the invalid one directly, like the "Invalid GranuleSpatialRepresentation enum" case above?  I'm sure there's a reason, just doesn't look intuitive to me.

Reply:
I wanted to test invalid CoordinateSystems. `The assert-invalid-spatial` function accepts GranuleSpatialRepresentation as a parameter but not a CoordinateSystem (although it is labeled coord-sys as the parameter for some reason). The assoc-in was to just avoid having to create another geometry variable with an invalid CoordinateSystem but if that is easier to read I could add another one.

Reply:
I think you should add one if you want to test CoordinateSystem too...  which brings another question - the ticket only says to "Add validation for GranuleSpatialRepresentation from ISO MENDS".  If you are adding the validation for CoordinateSystem too, could you update the ticket?

Reply:
done